elasticsearch/class-health-command.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Automattic\VIP\Elasticsearch;
+
+use \WP_CLI;
+use \WP_CLI\Utils;
+use \WP_Query as WP_Query;
+use \ElasticPress\Indexables as Indexables;
+use \ElasticPress\Elasticsearch as Elasticsearch;
+
+/**
+ * Helper commands to manage VIP Go Elasticsearch indexes
+ *
+ * @package Automattic\VIP\Elasticsearch
+ */
+class Health_Command extends \WPCOM_VIP_CLI_Command {
+
+	/**
+	 * @var WP_CLI\Fetchers\Plugin Plugin fetcher
+	 */
+	protected $plugin_fetcher;
+
+	public function __construct() {
+		require_once ABSPATH . 'wp-admin/includes/plugin.php';
+
+		parent::__construct();
+
+		$this->plugin_fetcher = new \WP_CLI\Fetchers\Plugin();
+	}
+
+	/**
+	 * Validate DB and ES index post counts
+	 *
+	 * ## OPTIONS
+	 *
+	 *
+	 * ## EXAMPLES
+	 *     wp vip-es health validate-counts
+	 *
+	 * @subcommand validate-counts
+	 */
+	public function validate_counts( $args, $assoc_args ) {
+		// Make sure ElasticPress is installed and active
+		$this->is_elasticpress_active();
+
+		// Get indexable objects
+		$indexable = Indexables::factory()->get( 'post' );
+
+		$post_types = $indexable->get_indexable_post_types();
+
+		WP_CLI::line( sprintf( "Checking %d post types (%s)\n", count( $post_types ), implode( ',', $post_types ) ) );
```

Comment:
```suggestion
		WP_CLI::line( sprintf( "Checking %d post types (%s)\n", count( $post_types ), implode( ', ', $post_types ) ) );
```

For readability, probably should add a space to the `implode()`.